File: S5/T2/Main.java
import java.util.*;
import java.util.Scanner;
public class Main{

  void solution(String[] array) {
      
    // Write code here
   
    
//     print new array here
        System.out.println(Arrays.toString()) // type name of new array inside Arrays.toString()
   
  }

  //Test code
  //Don't change anything below this line
  public static void main(String[] args){
    Main test = new Main();
    Scanner sc = new Scanner(System.in);
    int n = sc.nextInt();
    String[] array = new String[n]; 
    Scanner sc1 = new Scanner(System.in);
    for(int i=0; i<n; i++)  
    {  
         
        array[i]=sc1.nextLine();  
    }  
    test.solution(array);
  }
}

File: S5/T2/main.py
# Given function
def countLowerCase(m):
  count = 0
  for i in range(0, len(m)):
    if(m[i]>='a' and m[i]<='z'):
      count = count + 1
  return ("LowerCase=" + str(count))
  

# Test code
# Don't change anything in this block
if __name__ == "__main__":
  # For list of integers
  lst1 = []  
  n = int(input())
  lst1 = [int(item) for item in input().split()]
  
  
  
# Write code here 
# and call your function 
